docs/usage.md

```diff
@@ -38,10 +38,10 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run main.nf --input ./samplesheet.csv --outdir ./results -profile singularity
+nextflow run main.nf -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
```

Comment:
use `nextflow run phac-nml/snvphylnfc`

Reply:
Updated in 887e53de4eb3cbbd83f6cb0d881e4bbd1e26e931